tests/test_otel.py

```diff
@@ -288,3 +288,19 @@ def test_custom_resource_attributes(client, trace_service):
     assert get_attr(batch.resource, "service.name") == "test_service"
     assert get_attr(batch.resource, "my.name") == "my name"
     assert get_attr(batch.resource, "my.service") == "my service"
+
+
+@pytest.mark.parametrize(
+    "nginx_config",
+    [
+        {
+            "endpoint": """https://localhost:14318;
```

Comment:
Let's use something like "exporter_opts" to set cert instead of abusing "endpoint".

Reply:
Updated.